CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 0.23.6 -- 2019-06-16
```

Comment:
Please leave this in a "Unreleased" section, version 0.23.6 has not yet been released. ;)

Reply:
Oops, I don't know what I was thinking. I hope I got it right this time.